### PR TITLE
244 change table toolbar behavior

### LINKED ISSUE #244: Move folder description below table toolbar

### PR DESCRIPTION
**Pull request by swouf**
resolves #244

### CODE REVIEW

**Comment by swouf** [comment]
I cleaned the code. Only one test failed with this output `App should request context`.

File: src/components/main/TableToolbar.js [hunk 1]
```diff
@@ -53,63 +53,66 @@ DefaultActions.propTypes = {
...
+              className={classes.title}
+              color="inherit"
+              variant="subtitle1"
+              component="div"
+            >
+              {t('nbitem selected', { numSelected })}
+            </Typography>
+          ) : (
+            <>
+              <Typography
```
Comment (pyphilia) [inline src/components/main/TableToolbar.js:89]: remove comments?

**Review by pyphilia — CHANGES_REQUESTED** [review]
I think you left one commented section^^ Also could you rebase your branch on master? I see your branch also includes the app update one, which we don't want to merge yet! If you don't know about rebasing, we can have a call so I can show you.

File: src/components/main/TableToolbar.js (hunk 1 above)
Comment (swouf) [reply src/components/main/TableToolbar.js:89]: Sorry. I removed this. I'll check the other files too.

**Comment by swouf** [comment]
I rebased the branch. @pyphilia, let me know if it's ok for you now.

**Pull request merged**